WTWSMS/events/WtWSMS_zusk_misc_events.txt

```diff
@@ -606,7 +628,7 @@ character_event = {
 	hide_window = yes
 	
 	min_age = 6
-	max_age = 10
+	max_age = 15
```

Comment:
I thought sixteen would include after their education finished. I wanted to make sure that it would only happen before they reach the age of majority.